packages/backend/src/scheduler/SchedulerTask.ts

```diff
@@ -1444,6 +1446,19 @@ export default class SchedulerTask {
                 status: SchedulerJobStatus.COMPLETED,
             });
         } catch (e) {
+            if (
+                `${e}`.includes('invalid_grant') ||
+                `${e}`.includes('Requested entity was not found')
+            ) {
+                console.warn(
+                    `Disabling scheduler with non-retryable error: ${e}`,
+                );
+                await this.schedulerService.setSchedulerEnabled(
```

Comment:
after this runs, should we still cascade the error?

Reply:
Good question, probably not, this is what https://github.com/lightdash/lightdash/issues/10298 is supposed to achieve, right ?

Reply:
`Perhaps we should not say jobs have "failed" for some types of errors`

Great!